src/UI/examples/Toast/Standard/with_delay_time.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace ILIAS\UI\examples\Toast\Standard;
+
+function with_delay_time() : string
```

Comment:
Please remove this example, we will not need it anymore since consumers can't set that property anymore.